clientsession.go

```diff
@@ -563,6 +563,34 @@ func (s *ClientSession) SetClient(client *Client) *Client {
 	return prev
 }
 
+func (s *ClientSession) sendOffer(client McuClient, sender string, streamType string, offer map[string]interface{}) {
```

Comment:
Note to self: de-duplicate with https://github.com/danxuliu/nextcloud-spreed-signaling/blob/88636889d06639c8ca7810fd06e201b823ba9b97/hub.go#L1822 once this has landed.